added PII avoidance statement to support bundle docs (#216)
aws/eks-anywhere: docs/content/en/docs/tasks/troubleshoot/_supportbundle.md

`eksctl anywhere generate support-bundle-config` will generate a Support Bundle config yaml file for you to customize.

Do not add personally identifiable information (PII) or other confidential or sensitive information to your support bundle.

> If you provide the support bundle while obtaining support from AWS, it will be accessible by other AWS services in addition to AWS Support.

Reviewer: ```suggestion
When you obtain support from AWS, your support bundle may be accessible to other AWS services.
```

If you were given this wording, that if fine, but I thought this was a bit more clear.

Author: yeah it now reflects the wording I was given

> If you provide the support bundle while obtaining support from AWS, it will be accessible by other AWS services in addition to AWS Support.

Author: ```suggestion
If you provide the support bundle to get support from AWS, it will be accessible to other AWS services, including AWS Support.
```
